openpype/hosts/houdini/api/creator_node_shelves.py

```diff
@@ -35,11 +35,11 @@
 
 CREATE_SCRIPT = """
 from openpype.hosts.houdini.api.creator_node_shelves import create_interactive
-create_interactive("{identifier}", **kwargs)
+create_interactive("{identifier}", "{variant}", **kwargs)
 """
 
 
-def create_interactive(creator_identifier, **kwargs):
+def create_interactive(creator_identifier, default_variant, **kwargs):
```

Comment:
From the usage, this should be just variant, why `default_variant`?
```suggestion
def create_interactive(creator_identifier, variant, **kwargs):
```

Reply:
Actually it is _just_ the default value for the prompt - the user is still free to change the value. Whether that makes the argument clearer or not I'll leave in the middle for you both to discuss.

Reply:
correct yeah, `default_variant` represents better what the value is used for